content/guides/models/track/log/_index.md

```diff
@@ -81,4 +81,8 @@ wandb.log({"loss": 0.314, "epoch": 5,
 1. **Compare the best accuracy**: To compare the best value of a metric across runs, set the summary value for that metric. By default, summary is set to the last value you logged for each key. This is useful in the table in the UI, where you can sort and filter runs based on their summary metrics, to help compare runs in a table or bar chart based on their _best_ accuracy, instead of final accuracy. For example: `wandb.run.summary["best_accuracy"] = best_accuracy`
 2. **Multiple metrics on one chart**: Log multiple metrics in the same call to `wandb.log`, like this: `wandb.log({"acc'": 0.9, "loss": 0.1})` and they will both be available to plot against in the UI
```

Comment:
This and the next entry have incomplete sentence in bold, but the first and 4th have complete  cta-type sentences. Maybe "View multiple metrics in a single chart" and "Customize the X axis" or something like that?